File: semester-5_replit/oop_task-2/Metropolis.java
class Metropolis extends Domain {
  public Metropolis(String metropolisName) {
    super("unknown");
    this.metropolisName = metropolisName;
  }
  protected void getInfo() {
    System.out.println(domainName);
    System.out.println(metropolisName);
    System.out.println();
  }
  protected String metropolisName;
}

File: semester-5_vs/oop_lab-2/task-1_cpp/Metropolis.cpp
#include "Metropolis.h"

Metropolis::Metropolis(std::wstring name):Domain(L"unknown") {
  this->metropolisName = name;
}

void Metropolis::getInfo() {
  std::wcout << this->domainName << L"\n\r";
  std::wcout << this->metropolisName << L"\n\r";
  std::wcout << L"\n\r";
}
